tests/chainer_tests/test_function_node.py

```diff
@@ -869,6 +869,29 @@ def test_length_check(self):
             chainer.grad([y], [x], [None], [None, None])
 
 
+class TestUnchainSplit(unittest.TestCase):
```

Comment:
Since the test is somewhat corner-case, I'd place this test after the basic cases.  Could you move it to just after `TestGradComplex` or just before `TestGradV3Compat1`?

Could you rename the class?
```suggestion
class TestGradUnchainSplit(unittest.TestCase):
```

Reply:
Alright.